README.md

```diff
@@ -96,6 +96,12 @@ Thanks!
 
 ## High-level behavior
 
+Choise an adapter from [adapters](lib/active_model/serializer/adapter):
```

Comment:
Good catch @NullVoxPopuli !
I fixed it 🔧 https://github.com/rails-api/active_model_serializers/pull/1758/commits/90cfa92b5102dd11f7755b427b21318602279061